conftest.py

```diff
@@ -47,19 +47,29 @@ def pytest_addoption(parser):
 
 
 def pytest_configure(config):
-    # Override the markexpr argument to pytest
-    # See https://docs.pytest.org/en/latest/example/markers.html for more details
-    markexpr = []
-    if not config.option.large and not config.option.large_only:
-        markexpr.append("not large")
-    elif config.option.large_only:
-        markexpr.append("large")
-    if not config.option.lazy_import:
-        markexpr.append("not lazy_import")
-    if not config.option.requires_ssh:
-        markexpr.append("not requires_ssh")
-    if len(markexpr) > 0:
-        setattr(config.option, "markexpr", " and ".join(markexpr))
+    # Register markers to suppress `PytestUnknownMarkWarning`
+    config.addinivalue_line("markers", "large")
+    config.addinivalue_line("markers", "requires_ssh")
+    config.addinivalue_line("markers", "lazy_import")
+    config.addinivalue_line("markers", "notrackingurimock")
+    config.addinivalue_line("markers", "allow_infer_pip_requirements_fallback")
```

Comment:
pytest raises the following warning for unregistered markers:

```
/home/runner/work/mlflow/mlflow/tests/pyfunc/test_model_export_with_class_and_artifacts.py:650: PytestUnknownMarkWarning:
Unknown pytest.mark.large - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
```

Reply:
This warning is actually not harmful, but makes it difficult to find other warnings.